### PR TITLE
it-IT objects translations

### PR DESCRIPTION
**Pull request by LucaRed**
it-IT objects translations

### CODE REVIEW

File: objects/official/wall/official.mg-prar/object.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
             "en-GB": "Wall with Passageway",
             "de-DE": "Mauer mit Durchgang",
             "hu-HU": "Fal átjáróval",
+            "it-IT": "Muro con passaggio"
```
Comment (tupaschoal) [inline objects/official/wall/official.mg-prar/object.json:57]: Missing a comma

**Review by tupaschoal — COMMENTED** [review]
I guess you should be doing this pr on Localisation now, as we've merged the objects there .

**Comment by AaronVanGeffen** [comment]
Yep. The idea is that only functional changes are PR'ed to this repository now. There may be some old references to this repo in instructions that are still left to update though…

**Comment by Gymnasiast** [comment]
Since we changed the workflow very recently, I have gone ahead and applied your changes to the Localisation repositories, where the object translations now reside: https://github.com/OpenRCT2/Localisation/commit/66c14e700b7b79beb8e599d2c8213a21f5b15350

That commit should explain everything.

**Pull request closed without merging**